react/features/participants-pane/components/web/MeetingParticipants.tsx

```diff
@@ -103,6 +103,7 @@ function MeetingParticipants({
     const askUnmuteText = t('participantsPane.actions.askUnmute');
     const muteParticipantButtonText = t('dialog.muteParticipantButton');
     const isBreakoutRoom = useSelector(isInBreakoutRoom);
+    const { count: visitorsCount } = useSelector((state: IReduxState) => state['features/visitors']);
```

Comment:
This is a bad practice for `useSelector`. When you return the whole object from `useSelector` the component will re-render for any change in the object (not just the count). I know that now the object only has count and enabled but it's still a bad practice. Instead do
`const visitorsCount = useSelector((state: IReduxState) => state['features/visitors'].count);`

Reply:
fixed